src/lang/en-US/AdminPage.ts

```diff
@@ -0,0 +1,179 @@
+
+    'admin.grants.confirmation.alert': `Access to all items will be revoked and this action cannot be undone. Please review the list to continue.`,
+
+    'admin.users.cta.prefixInvitation': `Manage Invitations`,
+    'admin.users.prefixInvitation.header': `Manage Invitations`,
+    'admin.users.prefixInvitation.message': `This is a placeholder for a description.`,
+    'admin.users.prefixInvitation.label.capability': `Capability`,
+    'admin.users.prefixInvitation.label.type': `Type`,
+    'admin.users.prefixInvitation.cta.generateLink': `Generate Invitation`,
+    'admin.users.confirmation.listItem': `{identifier} with {capability} access.`,
+
+    'admin.prefix.cta.issueGrant': `Grant Access`,
+    'admin.prefix.issueGrant.header': `Share Data`,
+    'admin.prefix.issueGrant.message': `This is a placeholder for a description.`,
+    'admin.prefix.issueGrant.label.capability': `Capability`,
+    'admin.prefix.issueGrant.label.sharedPrefix': `Shared Prefix`,
+    'admin.prefix.issueGrant.label.sharedWith': `Shared With`,
+    'admin.prefix.issueGrant.cta.generateGrant': `Grant Access`,
+    'admin.prefix.issueGrant.notification.success.title': `Grant Created Successfully`,
+    'admin.prefix.issueGrant.notification.success.message': `{objectRole} has been shared with {subjectRole}.`,
+    'admin.prefix.issueGrant.error.invalidPrefix': `The value entered in the Shared Prefix text field is not an extension of an existing prefix.`,
+    'admin.prefix.confirmation.listItem': `{subjectRole} having {capability} over {objectRole}.`,
+    'admin.prefix.issueGrant.error.duplicatePrefix': `The requested access has already been provisioned.`,
+
+    'admin.alerts.header': `Organization Notifications`,
+    'admin.alerts.table.aria.label': `Organization Notifications Table`,
+    'admin.alerts.cta.addAlertMethod': `Configure Notifications`,
+    'admin.alerts.dialog.description': `Choose where you'd like notifications to be sent. To add an email address, select from the list of admin user emails or enter custom email addresses as a comma separated list.`,
+    'admin.alerts.dialog.emailSelector.inputError': `One or more emails are not formatted properly.`,
+    'admin.alerts.dialog.generate.header': `Configure Notification Methods`,
+    'admin.alerts.dialog.update.header': `Update Notification Methods`,
+    'admin.alerts.table.filterLabel': `Filter by Prefix or Email`,
+    'admin.alerts.table.noContent.header': `No subscriptions found.`,
+    'admin.alerts.table.noContent.message': `To begin receiving email notifications for a prefix you admin, click "Configure Notifications" above to create a subscription.`,
+    'admin.alerts.table.label.alertMethod': `Notification Method`,
+
+    'admin.tabs.users': `Account Access`,
+    'admin.tabs.connectors': `Connectors`,
+    'admin.tabs.api': `CLI-API`,
+    'admin.tabs.billing': `Billing`,
+    'admin.tabs.settings': `Settings`,
+
+    // Storage Mappings
```

Comment:
This is an example of how stuff was grouped with the object name made human readable as a comment.